.fastly/config.toml

```diff
@@ -4,13 +4,13 @@ config_version = 2
 api_endpoint = "https://api.fastly.com"
 
 [language]
-  [language.go]
-    tinygo_constraint = ">= 0.24.0-0" # NOTE -0 indicates to the CLI's semver package that we accept pre-releases (TinyGo users commonly use pre-releases).
-    toolchain_constraint = ">= 1.17"
+[language.go]
+tinygo_constraint = ">= 0.26.0-0" # NOTE -0 indicates to the CLI's semver package that we accept pre-releases (TinyGo users commonly use pre-releases).
+toolchain_constraint = ">= 1.17"
```

Comment:
TinyGo 0.26 dropped support for Go 1.17: https://github.com/tinygo-org/tinygo/releases/tag/v0.26.0

Reply:
Ah! OK cool will update that. Thanks!